desktop/test/run-visdiff.js

```diff
@@ -0,0 +1,185 @@
+import process from 'process'
+import fs from 'fs'
+import path from 'path'
+import {execSync} from 'child_process'
+import _ from 'lodash'
+import gm from 'gm'
+import github from 'octonode'
+
+const BUCKET_S3 = 's3://keybase-app-visdiff'
+const BUCKET_HTTP = 'http://keybase-app-visdiff.s3.amazonaws.com'
+const MAX_INLINE_IMAGES = 6
+
+const DIFF_NEW = 'new'
+const DIFF_REMOVED = 'removed'
+const DIFF_CHANGED = 'changed'
+const DIFF_SAME = 'same'
+
+function renderScreenshots (commitRange) {
+  for (const commit of commitRange) {
+    console.log(`Rendering screenshots of ${commit}`)
+    execSync(`git checkout -f ${commit} && mkdir -p screenshots/${commit} && npm run render-screenshots -- screenshots/${commit}`)
+  }
+  execSync(`git checkout -f ${commitRange[1]}`)
+}
+
+function compareScreenshots (commitRange, diffDir, callback) {
```

Comment:
After looking into this, I think it's cleaner to keep as-is, since there's multiple cases where I use a `for...of` loop to run the same code on both commits.